src/Scheduling.Host/Jobs/PurchaseOrderRemindersScheduler.cs

```diff
@@ -58,21 +58,24 @@ protected override async Task ExecuteAsync(CancellationToken stoppingToken)
 
                     this.log.Info("Found " + deliveries.Count() + " eligible for a reminder email.");
 
-
                     if (deliveries.Count > 0)
                     {
-                        this.log.Info($"Sending ${deliveries.Count} Emails: ");
-
+                        var supplierGroups = deliveries.GroupBy(d => d.PurchaseOrderDetail.PurchaseOrder.SupplierId);
+                        var enumerable = supplierGroups.ToList();
+                        this.log.Info($"Sending ${enumerable.ToList().Count} Emails: ");
 
-                        foreach (var d in deliveries)
+                        foreach (var g in enumerable)
                         {
                             this.dispatcher.Dispatch(new EmailPurchaseOrderReminderMessageResource
                                                          {
-                                                             OrderNumber = d.OrderNumber,
-                                                             OrderLine = d.OrderLine,
-                                                             DeliverySeq = d.DeliverySeq,
-                                                             Timestamp = DateTime.Now,
-                                                             Test = true
+                                                            Deliveries = g.ToList().OrderBy(o => o.OrderNumber).Select(d => new PurchaseOrderDeliveryKey()
+                                                                {
+                                                                    OrderNumber = d.OrderNumber,
+                                                                    OrderLine = d.OrderLine,
+                                                                    DeliverySequence = d.DeliverySeq
+                                                                }),
+                                                            Timestamp = DateTime.Now,
+                                                            Test = true
```

Comment:
this Test = true flag means they still won't be sent to suppliers for real even after this PR is merged - they'll just get sent to myself so I can check a few over the next couple of days before I switch them on for real